generators/entity-server/templates/quarkus/src/test/java/package/_partials/get_all.ejs

```diff
@@ -31,8 +34,21 @@
             .then()
             .statusCode(CREATED.getStatusCode())
             .extract().as(ENTITY_TYPE);
+    <%_ if (hasPagination) { _%>
 
-        // Get all the <%= entityInstance %>List
+        // Compare the number of records returned with the number at the start of the test
+        assertThat(Integer.parseInt(
```

Comment:
try to keep assertation as "fluent" as possible (ie. assertation should look like be a plain English sentence).
Could you please extract the RESTAssured call into a local variable?